extensions/amp-selector/1.0/selector.type.js

```diff
@@ -25,6 +25,7 @@ var SelectorDef = {};
  * @typedef {{
  *   as: (string|PreactDef.FunctionalComponent),
  *   disabled: (boolean|undefined),
+ *   domElement: !Element,
```

Comment:
Can we add a comment before or after typedef declaration that `domElement` is only used for shimming.